### DISCUSSION TITLE
why CAFE5 does not show familes amount with no changes which is summarized in CAFE4

### DISCUSSION THREAD

**Discussion by ghost** [hahnlab/CAFE5#130] [Q&A] [answered]
Hi，I compare CAFE4 and CAFE5 and is curious about why CAFE5 does not show familes amount with no changes which is summarized in CAFE4.

**Comment by benfulton** [comment, marked as answer]
The CAFE4 report file is back in CAFE 5.1. It once again shows the families with no changes.
